lib/ex_unit/lib/ex_unit/doc_test.ex

```diff
@@ -232,12 +232,20 @@ defmodule ExUnit.DocTest do
   end
 
   defp filter_by_opts(tests, opts) do
-    only = opts[:only] || []
-    except = opts[:except] || []
+    except = Keyword.get(opts, :except, [])
 
-    tests
-    |> Stream.reject(&(&1.fun_arity in except))
-    |> Stream.filter(&(Enum.empty?(only) or &1.fun_arity in only))
+    case Keyword.get(opts, :only) do
```

Comment:
Could we replace this with `Keyword.fetch/2`?

Reply:
Much better! Thanks!

Reply:
Done in https://github.com/elixir-lang/elixir/pull/8108/commits/a900bbde0f52561bd470ae001335d246ef8ec1dd